python/packagedata/WallGoCollision/__init__.py

```diff
@@ -1 +1,3 @@
-from ._WallGoCollision import *
\ No newline at end of file
+from ._WallGoCollision import *
+
+__version__ = getVersionNumber()
```

Comment:
Can you explain this function please? It seems to be written in the `BindToPython.cpp`, which prints something called `WG_VERSION`. How does this relate to the `_version.py` file, and the git tag?

Reply:
I am partly asking because perhaps we should do something similar in the WallGo repo.

Reply:
WG_VERSION is the preprocessor constant passed from CMake (where it's parsed from the git tag). The other option would be to import __version__ from _version.py, which is automatically generated by setuptools_scm. I think for the WallGo package we should do this.

Here I use the preprocessor constant instead because this way we get a version number even if compiling manually with CMake.